src/lib/core/modules/coderunner/runCode.js

```diff
@@ -24,6 +27,9 @@ class RunCode {
   }
 
   registerVar(varName, code) {
+    // Disallow `eval` and `require`, just in case.
+    if(code === eval || code === require) return;
```

Comment:
If code is equal to the function? Not sure this makes sense

Reply:
That function can also receive values, as it is shown a few lines below. Or am I misreading it?

Reply:
this works